db/migrate/20201030002614_add_user_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToOrders < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :orders, :user, foreign_key: true
```

Comment:
Weird again as this migration should have been in my PR, odd that it isn't in main already. Gonna have to check everything when merged to be sure nothing went awry. I'll take that action since it looks like it might have been my PR gone wrong (if anything goes wrong that is).